README.md

```diff
@@ -33,7 +33,8 @@ This repository contains the open source Blob SDK for Go. The [File SDK][File SD
 |-----------------|---------------------------|----------------------------------------------------------|
 | 2016-05-31      | 0.2.0                     | github.com/Azure/azure-storage-blob-go/2016-05-31/azblob |
 | 2017-07-29      | 0.2.0                     | github.com/Azure/azure-storage-blob-go/2017-07-29/azblob |
-| 2018-03-28      | 0.3.0+                    | github.com/Azure/azure-storage-blob-go/azblob            |
+| 2018-03-28      | 0.3.0 - 0.5.0             | github.com/Azure/azure-storage-blob-go/azblob            |
+| 2018-11-09      | 0.6.0                     | github.com/Azure/azure-storage-blob-go/azblob            |
```

Comment:
seems like your branch was based on master. Please submit another PR (by pushing to a branch to this repo and triggering CI), and cherry pick your changes on top of dev for a cleaner view.